pvp_club/init.lua

```diff
@@ -320,3 +320,26 @@ mt.register_on_respawnplayer(function (player)
     })  
     player:set_hp(100)
 end)
+
+minetest.register_chatcommand("skills", {
```

Comment:
I think better to changing it to stats:

```suggestion
minetest.register_chatcommand("stats", {
```